aspnetcore/blazor/hybrid/tutorials/wpf.md

```diff
@@ -120,8 +59,24 @@ At the top of the project file, change the SDK to `Microsoft.NET.Sdk.Razor`:
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 ```
 
+Add the following [MSBuild `Content` item](/visualstudio/msbuild/common-msbuild-project-items#content) to the project file, which copies the contents of the `wwwroot` folder to the output directory without compiling the assets in the folder:
+
+```xml
+<ItemGroup>
+  <Content Update="wwwroot\**" CopyToOutputDirectory="PreserveNewest" />
+</ItemGroup>
+```
```

Comment:
This should no longer be necessary. Let me know if you're seeing otherwise!

```suggestion
```

Reply:
I added it because it was still in your old GH example. I'll yank it locally to confirm it's ok to remove ... and then 🔪 it.

Reply:
Ok, it's good to 🔪 it.

What about in the WinForms version? Should it remain for that one, or should it be 🔪 there, too?

Reply:
The Content item group can be removed from the Windows Forms tutorial as well.

Reply:
Ok, thx ... I'll take care of it right now.